Dockerfile

```diff
@@ -16,7 +16,7 @@ WORKDIR /build
 # This step installs all the build tools we'll need
 RUN apk update && \
     apk upgrade --no-cache && \
-    apk add --no-cache git make g++ alpine-sdk openssl ncurses-libs wget ca-certificates yaml-dev python nodejs-npm bash erlang-crypto libssl1.1
+    apk add --no-cache make g++ git openssl-dev nodejs-npm python yaml-dev
```

Comment:
Why do we need `python` here?

Reply:
I think `node-sass` needs it in the `npm install` step for the webapp

Reply:
^this